Documentation/block.md

```diff
@@ -35,6 +35,10 @@ metadata:
 provisioner: rook.io/block
 parameters:
   pool: replicapool
+  # Specify the Rook cluster from which to create volumes.
+  # If not specified, it will use `rook` as the name of the cluster.
+  # i.e. if you are in a different namespace this must be set to that namespace's name
+  #clusterName: rook  
```

Comment:
how about if you uncomment the line with `clusterName: rook` and remove the other contents? This seems like it will be obvious enough, and keep the sample simple. It will also be more obvious after the param is renamed with #1614 that @rohan47 is working on.